themes/default/content/docs/intro/pulumi-service/org-tokens/_index.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "Organization Access Tokens"
+meta_desc: Overview of the Organization Access Tokens feature and how to use it.
+menu:
+  reference:
+    weight: 4
+---
+Organization Access Tokens provide Enterprise Growth and Business Critical customers the opportunity to manage resources and stack operations for their organization independent of a single-user account. This prevents organizations from needing to pass around a Personal Access Token belonging to a privileged organization member or admin in their automated tooling. This eliminates risk with modifying or removing organization memberships, as members can be decoupled from such automation pipelines. This also promotes less privileged access, as an Organization Access Token, unlike a Personal Access Token, is granted privileges only to the organization in which it was created, rather than to all organizations a single user belongs to.
+
+Tokens are created, managed, and removed by organization administrators, accessible on the Organization Settings page.
+
+## Creating an Organization Access Token
+
+From the organization’s homepage, visit Settings, then Access Tokens. To be shown this menu option, you must be on an Enterprise Growth or Business Critical subscription or trial. From the token landing page, you will see a “Create token” button in the top right of the page. Click this button to be prompted to create a token.
+
+The token must have a name that is unique among all Organization Access Tokens assigned to it. This allows tokens taking operations on behalf of your organization to be identifiable in the event that one is compromised. This name must be unique even among deleted tokens, in order to maintain the integrity of Audit Log Events which persist the token’s name (even for a deleted token) as part of the event (see below). Any other organization admin can delete this token; it is not owned by the admin which created it.
+
+Creation of any Organization Access Token, and the user who performed it, is logged as an Audit Log Event.
+
+## Viewing Organization Access Tokens
+
+Organization Access Tokens are viewed by going to the organization’s Settings page, then selecting Access Tokens from the menu. As with Personal Access Tokens, this table displays all tokens belonging to your organization, and at what time they were last successfully used to carry out an operation. Only organization admins of an organization with an Enterprise Growth or Business Critical subscription will see access to this page.
+
+## Deleting an Organization Access Token
+
+An access token can be deleted by any organization admin at any time. Click the ellipsis icon at the end of the row, and select “Delete token”.
+<img src="./ellipsis-delete-token.png">
+
+You will be asked to confirm. If you choose to delete a token, its access will immediately be revoked and all further operations using it will fail as unauthorized.
+
+## Audit Logs
```

Comment:
id recommend using a more specific heading, something like `Organization Token Audit Logs`
same for permissions, and API access

alternatively if you can make these h3s in a section then you could maybe drop that addition

Reply:
im worried about someone searching for audit logs for example in our docs search and this being a result, and maybe not what they are looking for